astroquery/ned/tests/data/query_references.xml

```diff
@@ -14,7 +14,7 @@ under cooperative agreement AST0122449 with The Johns Hopkins University.
 </DESCRIPTION>
```

Comment:
test data should not be modified unless NED is really returning something different now.  Is it?  If not, best revert this change.  If so, maybe this belongs in a different PR?